kumascript/macros/Interwiki.ejs

```diff
@@ -9,6 +9,9 @@
 var path = $1;
 var title = $2 || path;
 
+// Uppercase first character in path, replace all spaces with underscores.
+path = ($1.charAt(0).toUpperCase() + $1.slice(1)).replace(/ /g, "_");
```

Comment:
Drive-by comment, but 
```suggestion
path = (path.charAt(0).toUpperCase() + path.slice(1)).replace(/ /g, "_");
```
otherwise that initialization of the variable on line 9 is for nothing.

Reply:
> otherwise that initialization of the variable on line 9 is for nothing.

It still causes the title to be set to the `$1` value if there’s no `$2`. But then arguably line 10 should just be `var title = $2 || $1` and line 13 I added could be moved up to line 9. I wonder if I should amend the patch to make it do that?